specification/automanage/resource-manager/Microsoft.Automanage/preview/2021-04-30-preview/automanage.json

```diff
@@ -1283,6 +1283,52 @@
           "application/json"
         ]
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Automanage/listServicePrincipalId": {
+      "post": {
+        "tags": [
+          "ServicePrincipalId"
+        ],
+        "operationId": "Automanage_ListServicePrincipalId",
+        "description": "Returns the Automanage ADD first party Application Service Principal Id for the subscription.",
+        "x-ms-examples": {
+          "List service principal id": {
+            "$ref": "./examples/listServicePrincipalId.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
>200 [](http://example.com/codeflow?start=11&length=3)

x-ms-long-running-operation needs to return 202 for async response.

Reply:
Thank you! I will check more about this.

In Microsoft.OperationalInsights operations "POST/SUBSCRIPTIONS/RESOURCEGROUPS/PROVIDERS/MICROSOFT.OPERATIONALINSIGHTS/WORKSPACES/LISTKEYS", we return 200 and we followed the same pattern and used 200. 

Please check the below Kusto Query screenshot.

![image](https://user-images.githubusercontent.com/16600949/146502646-4b068186-950d-4cb6-b1b2-7bf0b8e680ab.png)

Reply:
**x-ms-long-running-operation :  false** is not required. I should have removed it earlier.  Please review the current one.